packages/ckeditor5-media-embed/src/mediaembedediting.js

```diff
@@ -76,6 +80,19 @@ export default class MediaEmbedEditing extends Plugin {
 						);
 					}
 				},
+				{
+					name: 'gif',
+					url: [
+						/(http(s?):)([/|.|\w|\s|-])*\.gif/
+					],
+					html: match => {
+						const url = match[ 0 ];
+
+						return (
+							`<img src=${ url } style="width: 100%;" >`
```

Comment:
@plsdev89 can we add an alt as well? Maybe capture the file name from the regex?

Reply:
Done